app/simulation/Form.tsx

```diff
@@ -40,7 +40,17 @@ function Form({ rules }) {
 
   const target = objectif ? decodeDottedName(objectif) : 'aides'
 
-  const engine = useMemo(() => new Publicodes(rules), [rules])
+  const engine = useMemo(
+    () =>
+      new Publicodes(rules, {
```

Comment:
Merci ! J'ignorais que ça existait. Ce serait quand même pas mal de garder les logs de warning mais sous forme condensée, du style "[n] warnings from publicodes : enable warning log to see why". Mais pas d'urgence, le code est clair, pour une autre fois.